core/drivers/clk/clk-stm32mp15.c

```diff
@@ -467,6 +476,7 @@ static const uint8_t rtc_parents[] = {
 static const struct stm32mp1_clk_sel stm32mp1_clk_sel[_PARENT_SEL_NB] = {
 	/* Secure aware clocks */
 	_CLK_PARENT(_STGEN_SEL, RCC_STGENCKSELR, 0, 0x3, stgen_parents),
+	_CLK_PARENT(_I2C35_SEL, RCC_I2C35CKSELR, 0, 0x7, i2c35_parents),
```

Comment:
this one should fall in the `/* Always non-secure clocks ... */` list right below.
Would you mind moving it and condition with `#ifdef CFG_WITH_NSEC_I2CS`?